cmd/puma-dev/main_linux.go

```diff
@@ -113,3 +116,53 @@ func main() {
 		log.Fatalf("Error listening: %s", err)
 	}
 }
+
+func parseEnvFlags() {
+	if !*fDebug {
+		envVal := os.Getenv("PUMA_DEV_DEBUG")
+		if envVal != "" {
+			*fDebug = true
+		}
+	}
+
+	if *fDomains == DefaultDomains {
+		envVal := os.Getenv("PUMA_DEV_DOMAINS")
+
+		if envVal != "" {
+			*fDomains = envVal
+		}
+	}
+
+	if *fHTTPPort == DefaultHttpPort {
+		envVal := os.Getenv("PUMA_DEV_HTTP_PORT")
+
+		if envVal != "" {
+			*fHTTPPort, _ = strconv.Atoi(envVal)
```

Comment:
We probably want to handle errors here, but I'm not sure what the most idiomatic way to surface those would be.